### ISSUE TITLE
Windows run XLSX fail 

### ISSUE THREAD

**Issue by joeshmoe189** [obsidianforensics/hindsight#36]
Hello all, 
So i'm haing a weird problem, when running the hindsight.py in my linux envoirment I will be able to use this tool normally, sadly I need this to run in windows. When I attempt to do so (through the command line)  with the same script and python versions etc. I believe the reason is the "error" is the line shown below. If anyone can help me out that would be awesome. 
No handlers could be found for logger "pyhindsight.analysis"

**Comment by obsidianforensics** [comment]
I don't think that error (about the logging) should stop Hindsight from running. Can you show the whole output (paths/etc redacted if needed)? 

Alternatively, on Windows you could try the exes (https://github.com/obsidianforensics/hindsight/releases/latest).
